libevmasm/PeepholeOptimiser.cpp

```diff
@@ -173,6 +173,34 @@ struct CommutativeSwap: SimplePeepholeOptimizerMethod<CommutativeSwap, 2>
 	}
 };
 
+struct SwapComparison: SimplePeepholeOptimizerMethod<SwapComparison, 2>
+{
+	static bool applySimple(AssemblyItem const& _swap, AssemblyItem const& _op, std::back_insert_iterator<AssemblyItems> _out)
+	{
+		static map<Instruction, Instruction> s_ops;
```

Comment:
Why not use initializer lists?

Reply:
Naming: Something like `opposite ops` or `swapped ops`?

Reply:
My clang warned that I cannot use initialiser lists there for some reason. Ah that was because by mistake I had it in the `struct` and not within the function. Can change.

Reply:
Fixed.